.github/workflows/release_branch_issue.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       - run: echo "VA_DUE_DATE=$(date -d "+6 days" '+%A %b %d, %Y')" >> $GITHUB_ENV
       - run: echo "RELEASE_DATE=$(date -d "+13 days" '+%A %b %d, %Y')" >> $GITHUB_ENV
       - uses: actions/checkout@v3
-      - run: echo "${{ secrets.GITHUB_TOKEN }}" >> token.txt
+      - run: echo "${{ secrets.GH_ACTIONS_PAT }}" > token.txt
```

Comment:
Is this PAT tied to your github account?

Reply:
no. It's not!